src/com/jambit/Database.java

```diff
@@ -0,0 +1,48 @@
+package com.jambit;
+
+import java.sql.*;
+
+public class Database {
+  private static final String HOST = "azubipi1.client.jambit.com";
+  private static final String USERNAME = "root";
```

Comment:
You should create a special user with access only to that schema

Reply:
Same here. The database is not fully set up at the moment. For testing purposes I used the login infos I was provided with.